$NetBSD: patch-libr-core-p-Makefile,v 1.1 2015/02/27 12:11:20 gdt Exp $

Cherry-picked from upstream a738fd259cb18bc03689c5377d37f7599078fdb1
"Enable support for --syszip in libr/core/p/Makefile"

--- libr/core/p/Makefile.orig	2014-07-02 16:26:02.000000000 +0000
+++ libr/core/p/Makefile
@@ -1,4 +1,5 @@
 include ../../config.mk
+include ../../../shlr/zip/deps.mk
 
 CFLAGS=-I../../include -Wall -shared -fPIC ${LDFLAGS_LIB} ${LDFLAGS_LINKPATH}..
 # XXX
@@ -12,7 +13,6 @@ CFLAGS+=-L../../syscall -lr_syscall -L..
 CFLAGS+=-L../../diff -lr_diff -L../../lang -lr_lang -L../../debug -lr_debug
 CFLAGS+=-L../../bp -lr_bp -L../../reg -lr_reg -L../../asm -lr_asm
 CFLAGS+=../../../shlr/sdb/src/libsdb.a
-CFLAGS+=../../../shlr/zip/librz.a
 CFLAGS+=../../../shlr/gdb/lib/libgdbr.a
 #CFLAGS+=-L../../anal -lr_anal -L../../core -lr_core
 LDFLAGS+=${LINK}
